docs/ralph/contracts.md

```diff
@@ -309,6 +307,57 @@ Contract Baz() implements Foo, Bar {
 }
 ```
 
+## Contract Method Calls
+
+In Ralph, there are two ways to call contract methods:
+
+1. method index: The index of the method in the contract code.
+2. method selector: Calculated from `hash("methodName(methodParamTypes)->(methodReturnTypes)")`, introduced in the Rhone upgrade. This allows a contract to inherit multiple interfaces.
+
+Calls made directly using the contract will use the method index, while calls using an interface will use the method selector, for example:
+
+```rust
+Interface IFoo {
+  pub fn foo() -> ()
+}
+
+Contract Foo() implements IFoo {
+  pub fn foo() -> () {}
+}
+
+let fooId = #...
+IFoo(fooId).foo() // This will use the method selector to call the contract
+Foo(fooId).foo()  // This will use the method index to call the contract
+```
+
+To maintain compatibility with contracts already deployed on the chain that do not support method selector calls, we introduced the `@using(methodSelector = true/false)` annotation. This annotation can only be applied to interfaces:
+
+```rust
+@using(methodSelector = false)
+Interface IFoo {
+  pub fn foo() -> ()
+}
+
+IFoo(fooId).foo() // This will not use the method selector to call the contract method, but will use the method index instead
+```
+
+In addition, Ralph also supports specifying the method index at compile time:
+
+```rust
+Interface IFoo {
+  @using(methodIndex = 2)
```

Comment:
Can we also specify the method index for Contract? E.g.
```
Contract Foo() implements IFoo {
  @using(methodIndex = 2)
  pub fn foo() -> () {}
}
```

Reply:
No. The main purpose of introducing `methodIndex` was to allow for easy calling of already deployed contracts using interfaces before the Rhone upgrade.

Reply:
Okay. Let's keep it like this